### PR TITLE
Add a close button ( #347 ) to the find and find in project panels

### PR DESCRIPTION
**Pull request by megametres**
Hello, a simple change to add a close button #347

I dont think that the "Close this panel with the esc key" message is a viable alternative to the close button.

### CODE REVIEW

**Comment by simurai** [comment]
I'm flip-flopping on this.

- Adding a close button is common and would help initially.
- Adding a close button would train people to use it, while there is an easier way.

Here the issue where the "Close this panel with the `esc` key" got added instead of a close button: https://github.com/atom/find-and-replace/issues/164

/cc @atom/feedback Any strong opinions about it?

**Comment by thomasjo** [comment]
For me this boils down to *affordance* (in the UX sense) — while we can train people to press esc, it is not truly intuitive, and also not discoverable. Hence poor affordance. The affordance of a close button on the other hand, is much better. It affords being pressed, and is immediately discoverable and intuitive.

**TL; DR:** I vote :+1: on this.

**Comment by thomasjo** [comment]
Also if we add the close button, we can teach people about the esc shortcut via a tooltip on the close button.

**Comment by simurai** [comment]
*If* the close button gets added, having the "Close this panel with the `esc` key" feels strange on the left. Also, the `x` could be confused with clearing the options and not closing the panel.

screen shot 2016-03-29 at 2 48 40 pm

- So yeah, maybe the "Close this panel with the `esc` key" with a tooltip on the `x`
- Center align the options

screen shot 2016-03-29 at 3 44 49 pm

**Comment by megametres** [comment]
I have another concern about the same topic, I'd like to change the shortcut ctrl+F to be on "Toggle Find in buffer" instead of "Find in buffer" and the same for ctrl+shift+F

It is also related to the closing of the panel, do anyone knows why it's not already this way and if there are any objections on the matter?

**Comment by 50Wliu** [comment]
> do anyone knows why it's not already this way

<kbd>Ctrl+F</kbd> is meant to _re-focus_ the panel.  Imagine the following scenario: You open <kbd>Ctrl+F</kbd> to search for something and find a result that you need to change, so you edit the text.  Then instead of having to manually click in the find box you can just <kbd>Ctrl+F</kbd> again to bring back focus to the find panel, and keep on searching.

**Comment by megametres** [comment]
I made the suggested adjustment with one detail, I find more useful/pretty the "Finding with option" aligned to the left
close-button

**Comment by elharo** [comment]
+1

**Comment by simurai** [comment]
Made a PR to move the labels back to the right: https://github.com/megametres/find-and-replace/pull/1

I guess this PR also needs a spec?

**Comment by maxbrunsfeld** [comment]
@megametres Thanks for your work on this. I'm going to close this out until you get a chance to address the remaining issues. FYI, we're planning on making some big changes to this package in order to stop using jQuery, so further conflicts may arise.
